packages/checks/test/index.ts

```diff
@@ -11,6 +11,8 @@ import { isString } from "../src/is-string";
 import { isSymbol } from "../src/is-symbol";
 import { isTransferable } from "../src/is-transferable";
 import { isTypedArray } from "../src/is-typedarray";
+import { isNil } from "../src/is-nil";
+import { isHexColorString } from "../src/is-hex-color-string";
```

Comment:
Can you please add these 2 modules to the re-exports in [the package's `index.ts`](https://github.com/thi-ng/umbrella/blob/master/packages/checks/src/index.ts)

Reply:
👍